[DOCS-2280][rum] document view renaming (#11176)
DataDog/documentation: content/en/real_user_monitoring/browser/modifying_data_and_context.md

- Reducing how much RUM data you're collecting, through sampling the data.
- Providing more context than what the default attributes provide about where the data is coming from.

## Override default RUM view names

> The RUM SDK generates a [view event][14] for each new page visited by your users. A view name is computed from the current page URL, with variable alphanumeric ids being removed automatically (for example, "/dashboard/1234" becomes "/dashboard/?").

Reviewer: ```suggestion
The RUM SDK automatically generates a [view event][14] for each new page visited by your users or when the URL changes (SPA). A view name is computed from the current page URL, with variable alphanumeric ids being removed automatically (for example, "/dashboard/1234" becomes "/dashboard/?").
```

Author: ```suggestion
The RUM SDK automatically generates a [view event][14] for each new page visited by your users, or when the page URL is changed (for single page applications). A view name is computed from the current page URL, with variable alphanumeric ids being removed automatically (for example, "/dashboard/1234" becomes "/dashboard/?").
```

> The RUM SDK automatically generates a [view event][14] for each new page visited by your users, or when the page URL is changed (for single page applications). A view name is computed from the current page URL, with variable alphanumeric ids being removed automatically (for example, "/dashboard/1234" becomes "/dashboard/?").

Reviewer 2: ```suggestion
The RUM SDK automatically generates a [view event][14] for each new page visited by your users, or when the page URL is changed (for single page applications). A view name is computed from the current page URL, where variable alphanumeric IDs are removed automatically—for example, "/dashboard/1234" becomes "/dashboard/?".
```

> Starting with [version 2.17.0][15], you may override the default view name with the `trackViewsManually` option:

Reviewer: ```suggestion
Starting with [version 2.17.0][15], you may specify your own view names by tracking view events manually with the `trackViewsManually` option:
```

1. Set up `trackViewsManually` to true when initializing RUM.
{{< tabs >}}
{{% tab "NPM" %}}

```javascript
import { datadogRum } from '@datadog/browser-rum';

datadogRum.init({
    ...,
    trackViewsManually: true,
    ...
});
```
{{% /tab %}}
{{% tab "CDN async" %}}
```javascript
DD_RUM.onReady(function() {
    DD_RUM.init({
        ...,
        trackViewsManually: true,
        ...
    })
})
```
{{% /tab %}}
{{% tab "CDN sync" %}}
```javascript
window.DD_RUM &&
    window.DD_RUM.init({
        ...,
        trackViewsManually: true,
        ...
    });
```
{{% /tab %}}
{{< /tabs >}}

> 2. Start views for each new page and define the associated view name. If not provided a name, the view name will default to the page URL path.

Reviewer: ```suggestion
2. Start views for each new page or route change and define the associated view name. If not provided a name, the view name will default to the page URL path.
```

Author: ```suggestion
2. Start views for each new page, or route change (for single page applications), and define the associated view name. If not provided a name, the view name defaults to the page URL path.
```

> 2. Start views for each new page, or route change (for single page applications), and define the associated view name. If not provided a name, the view name defaults to the page URL path.

Author: You **must** start views for each new page, or route change (for single page applications), and optionally define the associated view name (it defaults to the page URL path). No RUM data gets collected until the view is started.

> 2. You **must** start views for each new page, or route change (for single page applications), and optionally define the associated view name (it defaults to the page URL path). No RUM data gets collected until the view is started.

Reviewer 2: ```suggestion
2. You **must** start views for each new page or route change (for single page applications). You can optionally define the associated view name, which defaults to the page URL path. No RUM data is collected until the view is started.
```

{{< tabs >}}
{{% tab "NPM" %}}
```javascript
datadogRum.startView('checkout')
```

{{% /tab %}}
{{% tab "CDN async" %}}
```javascript
DD_RUM.onReady(function() {
    DD_RUM.startView('checkout')
})
```
{{% /tab %}}
{{% tab "CDN sync" %}}

```javascript
window.DD_RUM && window.DD_RUM.startView('checkout')
```
{{% /tab %}}
{{< /tabs >}}

> 
> **Note**: If you are using React, Angular, Vue or any other frontend framework, we recommend implementing the `startView` logic at the framework router level.

Reviewer: ```suggestion
**Note**: No events are collected until the first view is started.
**Note 2**: If you are using React, Angular, Vue or any other frontend framework, we recommend implementing the `startView` logic at the framework router level.
```

Author: Actually I think it's worth mentioning it as part of step 2, like:
"You **must** start views for each new page, or route change (for single page applications), and optionally define the associated view name (it defaults to the page URL path). No data gets collected until the view is started"

wdyt?

Reviewer: lgtm

> **Note**: If you are using React, Angular, Vue or any other frontend framework, we recommend implementing the `startView` logic at the framework router level.

Reviewer 2: ```suggestion
**Note**: If you are using React, Angular, Vue or any other frontend framework, Datadog recommends implementing the `startView` logic at the framework router level.
```
